### DISCUSSION TITLE
Source port distinction from remakes and clones

### DISCUSSION THREAD

**Discussion by nikuda** [opengaming/osgameclones#1558] [Ideas]
Add a key like `ports` to function like `remakes` and `clones` for commercial games that have been open sourced.

One of the last suggestions from #30 that was not yet done.

**Comment by JayFoxRox** [comment]
I think this should be even more precise. Here are some rough ideas:

- `implements` (an unofficial port, like OpenRCT2, largely based on reverse engineering)
- `ports` (just ported an existing open-source game to new platforms)
- `remakes` ("I implement what I see")
- `clones` ("I like this concept so I reproduce something similar with my own ideas")

Additionally: add `open-source` or `license` key as part of the "Originals", so if the game was open-source to begin with, or was open-sourced later, this is on record.

**Comment by cxong** [comment]
This should be a list. A game could for example be based on a port but contain gameplay features not found in the original, which means it's also a clone.

**Comment by cxong** [comment]
Based on #406 it might be better to stick this information in `info`. If you look at games which have drifted very far from their original source, e.g. all the open-source quake-engine-descended games, many of them have little in common with quake, so where their source code originated is more of a historical trivia that's not that important to players. What we have currently (`remake`, `clone`, `similar`) should be enough as far as players are concerned.

**Comment by cxong** [comment]
I'm keen to keep this issue moving forward. Tentatively closing for now, as this issue is blocking other games from being added. But we can always reopen this discussion and fix up the affected games later.
